src/js/components/uploadDetachedFiles/UploadDetachedFilesPage.jsx

```diff
@@ -380,41 +457,58 @@ export default class UploadDetachedFilesPage extends React.Component {
 								<p>{this.state.detachedAward.error.description}</p>
 							</div>;
 		}
+		else if(this.state.error == 1){
```

Comment:
Refactor this if the only difference in state 1, 2, and 3 is the text in the error message